.github/workflows/code_quality.yml

```diff
@@ -10,8 +10,13 @@ on:
         type: string
 
 jobs:
-  TestPython:
-    name: Code Quality
-    uses: ./.github/workflows/reuse_python_build.yml
-    secrets: inherit
-
+  Test:
+    name: Python
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        python-version: ['3.9', '3.10', '3.11']
```

Comment:
Mind adding 3.12? We have it in the other OpenJobDescription repos, but not this one for some reason.

Reply:
Fixed